### ISSUE TITLE
Accordion expand/collapse all aria-label is unclear

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#4531]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The Decision Reviews team received the following feedback in a 508 audit of the 0995 Supplemental Claim form, that uses the VA Accordion. About the Expand/Collapse all button for the VAAccordion:

> On the File A Supplemental Claim | Veterans Affairs screen (Step5 of 5Review application), under Step5 of 5Review application, the name of the button cannot be understood in relation to the state. For example, “Expand all accordions Button collapsed” is read. One possibility is to change the name, “Expand all accordions” or “Collapse all accordions”, without indicating the state. Another possibility is to keep the name the same, “all accordions Button collapsed” or “all accordions Button expanded”, and change the state—this is the convention used throughout the product. (New: 04/04/2025)

## What I expected to happen
See above 508 audit suggestions

## Reproducing
- CSS Library version: Not sure which one the comment was referring to back in 4/2025 - V3 of the library at least.
- Device: (e.g. iPhone 8, Macbook) Mac OS / VoiceOver
- Browser: (e.g. Firefox, IE 11) Chrome

Steps to reproduce:

1. Can be replicated on the storybook link for the component https://design.va.gov/storybook/?path=/docs/uswds-va-accordion--docs
2. Boot up screen reader and navigate to the expand or collapse all button and listen.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

### Details
Issue identified in a 508 audit, see second item https://github.com/department-of-veterans-affairs/va.gov-team/issues/107049

**Comment by jeana-adhoc** [comment]
My preliminary recommendation for this fix (Still need full DST review)

1. Remove `aria-expanded` from the “expand/collapse all” button. We should let text label alone communicate the action and not communicate a state on the button. Also - `aria-expanded` is not accurate on this button because the button itself is not in an expanded or collapsed state.
2. Add a more descriptive label and say `expand all sections` or `collapse all sections`. I don't want to use the word `accordions` because that's more technical/design speak that may lose context to unsighted users.
3. Add aria-live (polite) to announce the state change. 

Example code:

```
<div role="group" aria-label="Accordion controls">
  <!-- Note: no aria-expanded here -->
  <button id="toggle-all" type="button">
    Expand all sections
  </button>
</div>

<div id="accordion-status"
     role="status"
     aria-live="polite"
     aria-atomic="true"
     class="sr-only">
</div>
```

Next steps review with @amyleadem

**Comment by jeana-adhoc** [comment]
@amyleadem 's  [draft PR](https://github.com/department-of-veterans-affairs/component-library/pull/1745#issuecomment-3230000000)

We're still investigating the right solution.

Moving to next sprint.
